### PR TITLE
Danielle cantu

### PR DESCRIPTION
**Pull request by dlcantu**

### CODE REVIEW

**Comment by dlcantu** [comment]
Any API lab  and JS unit converter done. Django todo lab and JS todo lab WIP.

File: code/danielle/Any_API/any_api.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    })
+
+    function getData(number) {
+        fetch(`http://numbersapi.com/${number}/trivia`) // ${}
+            .then(function (response) {
+                console.log(response)
+                return response.text()
+            })
+            .then(function (data) {
+                console.log(data)
```
Comment (AlexDeMichieli) [inline code/danielle/Any_API/any_api.js]: @dlcantu this isn't outputting data on a page. Make sure to implement that part!

**Comment by dlcantu** [comment]
Any API lab completed

File: code/danielle/Any_API/any_api.js (hunk 1 above)
Comment (dlcantu) [reply code/danielle/Any_API/any_api.js]: It works now!

**Comment by Gage-Lieble** [comment]
Nice work! all that seems to be left is the JS todo lab

**Pull request merged**